### PR TITLE
[Chore] Cleaning up the component generator script

### PR DESCRIPTION
**Pull request by beau-charman**

### CODE REVIEW

File: scripts/componentBoilerplate/assets/Component.types.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
-import type { Themes } from '~/types';
+import type { Themes, ComponentTypeBoilerplate } from '~/types';
```
[the remark this answers is not in the document]
Comment (beau-charman) [reply scripts/componentBoilerplate/assets/Component.types.ts:1]: It is a template placeholder for this command https://github.com/aesop/aesop-gel/pull/468/files#diff-2ab06301109ad715587899866d259ee19870988c214dc21b5187d6510715ee04R41-R49

[the remark this answers is not in the document]
Comment (MkMan) [reply scripts/componentBoilerplate/assets/Component.types.ts:1]: Ah ... bash, my sworn enemy.

**Review by MkMan — APPROVED** [review]

**Review by MkMan — APPROVED** [review]
Can you please update the README to include instructions on how to use the script 🙂 ?

File: scripts/componentBoilerplate/assets/Component.types.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (beau-charman) [reply scripts/componentBoilerplate/assets/Component.types.ts:1]: we could make the 'variables' stand out a little more

**Comment by beau-charman** [comment]
I might conduct the readme updates in another PR, as there are a few changes I would like to make for now

File: scripts/componentBoilerplate/assets/Component.types.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (beau-charman) [reply scripts/componentBoilerplate/assets/Component.types.ts:1]: I could update all the variables to upper case? 

```
import type { Themes, _COMPONENT_TYPE_ } from '~/types';
```

[the remark this answers is not in the document]
Comment (MkMan) [reply scripts/componentBoilerplate/assets/Component.types.ts:1]: That's a good idea! Much easier to tell that way 👌

**Comment by beau-charman** [comment]
Sorry @MkMan , just asking for a quick re-review on the variables

**Pull request merged**